packages/botonic-plugin-google-analytics/src/index.js

```diff
@@ -0,0 +1,40 @@
+import Analytics from 'analytics'
+import googleAnalytics from '@analytics/google-analytics'
+
+export default class BotonicPluginGoogleAnalytics {
+  constructor({ botName, trackingId }) {
+    this.analytics = Analytics({
+      app: botName,
+      plugins: [
+        googleAnalytics({
+          trackingId: trackingId,
+        }),
+      ],
+    })
```

Comment:
You'll also need to call the `identify` method so that GA knows what user is sending the events. Something like
```
this.analytics.identify(session.user.id, {
  firstName: session.user.name,
  channel: session.user.provider,
  channel_id: session.user.provider_id
})
```
I'm not sure if you can set any params you want...
Also, I'm not sure if you can do this in the constructor or you'll nedd to do it in every `track` call.